helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -23,8 +23,10 @@ timeZone: Asia/Shanghai
 datadog:
   log:
     enabled: false
+    # besides the attributes you added, chart will append "source" and "service" attributes to the log attributes.
+    # please do not include the curly brackets in the value, like '"app": "starrocks", "tags": ["aa", "bb"]'.
     # see https://docs.datadoghq.com/containers/kubernetes/log/?tab=operator for more details.
-    tags: '[]'  # e.g. '["env:test"]'
+    logAttributes: ''
```

Comment:
doesn't look like easy to use for end user, if is hard to manipulate the json string in helm template, can we move it into operator, deserialize to json object and inject necessary attributes and serialize again.

Reply:
In https://docs.datadoghq.com/containers/kubernetes/log/?tab=operator,  they call it `attributes`.

Reply:
I think this integration work is better done in chart, because we just add some annotations. In Operator, we should add fileds to crd.
I will try to fix it to allow user use a standard json format.

Reply:
> In https://docs.datadoghq.com/containers/kubernetes/log/?tab=operator, they call it `attributes`.

They call the key-value in json `attributes` and also `tags` included.

The entire json is called LOG_CONFIG
```
ad.datadoghq.com/<CONTAINER_IDENTIFIER>.logs: '[<LOG_CONFIG>]'
```